src/Example/Contexts/SeedDb.cs

```diff
@@ -9,9 +12,74 @@ public static class SeedDb
     /// Database initialization
     /// </summary>
     /// <param name="serviceProvider">Service provider</param>
-    public static void Initialize(IServiceProvider serviceProvider)
+    public static async Task Initialize(IServiceProvider serviceProvider)
     {
         var context = serviceProvider.GetRequiredService<ExKirelIdentityContext>();
         context.Database.EnsureCreated();
+        
+        var adminRoleExist = true;
+        var roleManager = serviceProvider.GetRequiredService<RoleManager<ExRole>>();
+        var adminRole = await roleManager.FindByNameAsync("Admin");
+        if (adminRole == null)
+        {
+            adminRole = new ExRole(){ Name = "Admin", NormalizedName = "ADMIN" };
+            var result = await roleManager.CreateAsync(adminRole);
+            if (!result.Succeeded)
+                adminRoleExist = false;
+        }
+        var userRoleExist = true;
+        var userRole = await roleManager.FindByNameAsync("User");
+        if (userRole == null)
+        {
+            userRole = new ExRole(){ Name = "User", NormalizedName = "USER" };
+            var result = await roleManager.CreateAsync(userRole);
+            if (!result.Succeeded)
+                userRoleExist = false;
+        }
```

Comment:
I think you can unite this code in single function. Also, roleManager have RoleExistsAsync() method which can be used there